project/Dependencies.scala

```diff
@@ -50,7 +50,7 @@ object Dependencies {
     "com.github.pureconfig"        %% "pureconfig"                    % "0.12.3",
     "com.softwaremill.sttp.client" %% "async-http-client-backend-zio" % Versions.sttp,
     "com.softwaremill.sttp.client" %% "circe"                         % Versions.sttp,
-    "dev.zio"                      %% "zio-interop-cats"              % "2.0.0.0-RC14"
+    "dev.zio"                      %% "zio-interop-cats"              % "2.1.4.0-RC17"
```

Comment:
Since we upgraded branch to use zio 1.0.0, at the moment I'm writing this, cats interop hasn't been release. Note that the release will happen soonish, so please upgrade accordingly :).